src/clients/Deluge.ts

```diff
@@ -300,7 +365,8 @@ export default class Deluge implements TorrentClient {
 				message: `Failed to fetch torrent data: ${searchee.name} - (${searchee.infoHash})`,
 			});
 			logger.debug(e);
-			return { complete: false, save_path: "missing" };
+			throw new Error("web.update_ui: failed to fetch data from client");
```

Comment:
```suggestion
			throw new Error("web.update_ui: failed to fetch data from client", { cause: e });
```

Reply:
As with the other usage of cause, this isn't supported until we push to es2022 and will require a directive to suppress. 

I can add it if you still want to.

Reply:
yeah just throw in the directive. I'm gonna work on getting the tsconfig updated